R/utils.R

```diff
@@ -88,3 +88,34 @@ print_vrd_api <- function(x, ...){
 }
 
 
+#' clean_vrd_api()
+#'
+#'helper function for cleaning api database
+#'
+#' @param api_output api response to be cleaned
+#'
+#' @return dataframe
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' clean_vrd_api()
+#' }
+clean_vrd_api <- function(api_output){
+
+    # makes dataframe of output
+    df1 <- as.data.frame(unlist(api_output$content$ResultSet,recursive=FALSE))
+
+    # only keeps columns with values
+    df2 <- df1[, grep("Literal", colnames(df1))]
```

Comment:
here we are looking for columns that contain "Literal", and drop those that dont

what are the other options for columns if they dont have values?

Reply:
They have the data type as well as the name.  I just realized I need to fix the dataframe to output as numbers and dates still

Reply:
👍